CefSharp.OutOfProcess.Core/OutOfProcessHost.cs

```diff
@@ -37,6 +38,11 @@ private OutOfProcessHost(string outOfProcessHostExePath, string cachePath = null
             _cachePath = cachePath;
         }
 
+        public event EventHandler<DownloadCallbackDetails> DownloadCallback;
```

Comment:
i think we should switch form the event to a method called on the client. So we could call one specific client. The event would be handled by all clients which wouldn't scale well.

Reply:
also this event should be implemented explicitly to hide this event a bit more